### DISCUSSION TITLE
How to manually set certificate information (i.e., Server_Name, City, Org, etc)?

### DISCUSSION THREAD

**Discussion by MTBer06** [pivpn/pivpn#1393] [Feature requests]
Can these entries be changed from their defaults during initial installation and generation of the certificate authority and subsequent certificates?

**Comment by jellemdekker** [comment]
I've done some digging, because my curiosity was sparked by the `ca_info.txt` file in the repo. 
>PiVPN began as a rough merger of the code at [OpenVPNSetup](https://github.com/StarshipEngineer/OpenVPN-Setup).

([README.md](https://github.com/pivpn/pivpn/blob/master/README.md) from PiVPN)

In the original project, the user had the ability to change the certificate information exactly as you suggested:

> If you wish to enter identifying information for the CA, replace the default values in the file ca_info.txt (CO for country, ST for state/province/territory, ORG for organization, etc.) before executing the setup script; however, this is not required, and you may leave the ca_info.txt file as-is. After this, the script will prompt you in the command line for input in similar identifying information fields as it generates your server certificate. Enter whatever you like, or if you do not desire to fill them out, skip them by pressing enter; make sure to skip the challenge field and leave it blank.

([README.md](https://github.com/StarshipEngineer/OpenVPN-Setup/blob/master/README.md) from OpenVPN-Setup)

Since then, however, the line of code (#796) that used `ca_info.txt` in the certification generation process was modified by commit [0b96cae](https://github.com/pivpn/pivpn/commit/0b96cae1a37e8cce8c7656210022ec47627e6d41). Presumably to simplify the installation process, but let's ask @0-kaladin. So `ca_info.txt` can be deleted from the repo if I'm not mistaken.
